services/app-web/src/components/DownloadButton/DownloadButton.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Button, Link } from '@trussworks/react-uswds'
+
+type DownloadButtonProps = {
+    text: string
+    zippedFilesURL: string
+}
+
+export const DownloadButton = ({
+    text,
+    zippedFilesURL,
+}: DownloadButtonProps): React.ReactElement => {
+    return (
+        <div>
+            <Link href={zippedFilesURL} target="_blank">
```

Comment:
I don't think we want to nest a button inside a link, instead you can apply the button styles to the link like: 

`<Link className="use-button" variant="unsettled" >{text}</Link>`

Reply:
I tried to apply the styling right onto the link like this and it wasn't styling as a button, it kept things as a link for whatever reason.